federation/pkg/federation-controller/sync/controller.go

```diff
@@ -319,6 +319,13 @@ func (s *FederationSyncController) reconcile(namespacedName types.NamespacedName
 		return redeliver
 	}
 
+	if _, ok := s.adapter.(federatedtypes.SchedulerAdapter); ok {
+		reconcileStatus := s.updateFedSpec(obj)
+		if reconcileStatus != statusOK {
+			return reconcileStatus
+		}
+	}
+
 	return syncToClusters(
```

Comment:
I think you'll need a way to update the federated object after syncing to the clusters: the FRS controller sets the state of the FRS after syncing the cluster replica sets.